_members/cameron-mattson.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Cameron Mattson
+image: images/team/cameron_mattson.jpg
+description: Data Scientist
+role: BioData Scientist
```

Comment:
please make your role `programmer` (see https://github.com/WayScience/waysciencelab.com/blob/d005c611089ebbf5d8c245af1e72ccfa5299045c/_data/roles.yaml#LL17C6-L17C6)

Reply:
Just added the change